modules/registry-configuring-registry-storage-rhodf-cephfs.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/configuring-registry-storage-vsphere.adoc
+//
+// * registry/configuring_registry_storage/configuring-registry-storage-baremetal.adoc
+//
+// * registry/configuring_registry_storage/Configuring-the-registry-for-rhodf.adoc
+
+:_content-type: PROCEDURE
+[id="registry-configuring-registry-storage-rhodf-cephfs_{context}"]
+= Configuring the Image Registry Operator to to use CephFS storage with Red Hat OpenShift Data Foundation
+
+{rh-storage-first} integrates multiple storage types that you can use with the internal image registry:
+
+* Ceph, a shared and distributed file system and on-premises object storage
+* NooBaa, providing a Multicloud Object Gateway
+
+This procedure outlines the procedure to  configure the image registry to use CephFS storage.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the {product-title} web console.
+* You installed the `oc` CLI.
+* Persistent storage provisioned for your cluster, such as {rh-storage}.
+
+.Procedure
+
+. Create persistent volume claim (PCV) to use the `cephfs` storage class. For example:
```

Comment:
It is not recommended to use PVCs for the registry if there are other options are available. Do we need to document usage of cephfs as PVC?

Reply:
I didn't realize that, thanks! I can remove this procedure.